clients/ctl/admin-ui/src/features/taxonomy/TaxonomyTabContent.tsx

```diff
@@ -40,29 +61,66 @@ const TaxonomyTabContent = ({ useTaxonomy }: Props) => {
     return <Text>Could not find data.</Text>;
   }
 
+  const taxonomyType = labels.fides_key.toLocaleLowerCase();
+
   const handleSetEditEntity = (node: TaxonomyEntityNode) => {
     const entity = data?.find((d) => d.fides_key === node.value) ?? null;
     setEditEntity(entity);
   };
 
+  const handleSetDeleteKey = (node: TaxonomyEntityNode) => {
+    setDeleteKey(node.value);
+    onDeleteOpen();
+  };
+
+  const handleDelete = async () => {
+    if (deleteKey) {
+      const result = await onDelete(deleteKey);
+      if (isErrorResult(result)) {
+        toast(errorToastParams(getErrorMessage(result.error)));
+      } else {
+        toast(successToastParams(`Successfully deleted ${taxonomyType}`));
+      }
+      onDeleteClose();
+    }
+  };
+
   return (
-    <SimpleGrid columns={2} spacing={2}>
-      <AccordionTree
-        nodes={taxonomyNodes}
-        onEdit={handleSetEditEntity}
-        focusedKey={editEntity?.fides_key}
-      />
-      {editEntity ? (
-        <TaxonomyFormBase
-          labels={labels}
-          entity={editEntity}
-          onCancel={() => setEditEntity(null)}
-          onEdit={onEdit}
-          extraFormFields={extraFormFields}
-          initialValues={transformEntityToInitialValues(editEntity)}
+    <>
+      <SimpleGrid columns={2} spacing={2}>
+        <AccordionTree
+          nodes={taxonomyNodes}
+          onEdit={handleSetEditEntity}
+          onDelete={handleSetDeleteKey}
+          focusedKey={editEntity?.fides_key}
         />
-      ) : null}
-    </SimpleGrid>
+        {editEntity ? (
+          <TaxonomyFormBase
```

Comment:
I noticed that if I click "Edit" to show the form for a taxonomy item, then hit "Delete" on that same item (and confirm) the edit form will stick around for the deleted item. Attempting to "Update" the item will then error, of course. So maybe the call to `handleSetDeleteKey` needs to also clear the EditEntity.